### DISCUSSION TITLE
Welcome to Modern Web Discussions!

### DISCUSSION THREAD

**Discussion by daKmoR** [modernweb-dev/web#453] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

If you have issues with our products/packages then [issues](https://github.com/modernweb-dev/web/issues) are still the place but for everything else make yourself at home 🤗

**Comment by dogoku** [comment]
Just saying hi 👋

It's always fun when I discover a new project I like and I see @daKmoR as one of the contributors ❤️

**Reply by daKmoR** [reply]
hey there 👋
I hope you enjoy using Modern Web 🤗.

**Comment by ghost** [comment]
👋🏾 Hello! What I've read on both modern-web and open-wc, has resonated with me. I'm looking forward to learning more.
